core/src/main/java/org/elasticsearch/action/admin/indices/shrink/TransportShrinkAction.java

```diff
@@ -92,7 +92,12 @@ public void onResponse(IndicesStatsResponse indicesStatsResponse) {
                         return shard == null ? null : shard.getPrimary().getDocs();
                     }, indexNameExpressionResolver);
                 createIndexService.createIndex(updateRequest, ActionListener.wrap(response ->
-                    listener.onResponse(new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked())), listener::onFailure));
+                        listener.onResponse(
+                            new ShrinkResponse(response.isAcknowledged(), response.isShardsAcked(), updateRequest.index())
+                        ),
```

Comment:
The line breaks are awkward here, can we put each parameter to `createIndexService#createIndex` on a single line or wrap them a little differently?